### PR TITLE
Add support for negative dimensions in cumsum

### PR DESCRIPTION
**Pull request by abhudev**
Fixes #92329

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -2198,6 +2198,19 @@ def helper(dtype):
             self.assertEqual(e_string, "MPS does not support cumsum op with int64 input")
 
 
+    def test_cumsum_minus_one_axis(self):
```
Comment (kulinseth) [inline test/test_mps.py:2201]: Can you test the TestConsistency test for cumsum.

Comment (kulinseth) [reply test/test_mps.py:2201]: Also enable that for all types.

Comment (abhudev) [reply test/test_mps.py:2201]: TestConsistency passes, and expanded this test with datatypes (which all pass).

**Review by kulinseth — APPROVED** [review]

**Comment by DenisVieriu97** [comment]
@abhudev could you please take a look at the lint issues?

**Review by DenisVieriu97 — CHANGES_REQUESTED** [review]
Looks good - please take a look at the lint issues before merging

**Pull request merged**